test/dlc_tests/sanity/test_telemetry.py

```diff
@@ -304,6 +304,8 @@ def _run_tag_failure_IMDSv1_disabled(image_uri, ec2_client, ec2_instance, ec2_co
         ec2_client.delete_tags(Resources=[ec2_instance_id], Tags=[{"Key": expected_tag_key}])
 
     # Disable access to EC2 instance metadata
+    ec2_connection.run(f"sudo apt-get update -y")
+    ec2_connection.run(f"sudo apt-get install -y net-tools")
```

Comment:
reason for the above commands? can we install net-tools directly on the base ami if thats the requirement?

Reply:
as discussed, this is due to switch from ubuntu18 to ubuntu20.